content/quickstart_ocaml_project_dune.md

```diff
@@ -4,7 +4,9 @@ tags: [learning, quickstart]
 
 # Quickstart an OCaml app project using Dune
 
-(adapted from @bobbypriambodo's post [here](https://medium.com/@bobbypriambodo/starting-an-ocaml-app-project-using-dune-d4f74e291de8))
+(adapted from @bobbypriambodo's post [here](https://medium.com/@bobbypriambodo/starting-an-ocaml-app-project-using-dune-d4f74e291de8) )
+
+Apr 3, 2018
```

Comment:
I'm not sure that adding the date is a good idea.

Reply:
When looking at (OCaml) articles as a totally OCaml beginner I've been often trapped by, say old ocamlbuild or Oasis article, that were totally outdated ant that I took for an oasis. It wasn't.
In the future, we can consider a lifecycle for the OCamlverse articles in sync with OCaml and its tools. So no date on articles, just up-to-date stuff.

Reply:
I'd prefer that you remove the date. It's incongruous. Thanks.

Reply:
Done